src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -259,11 +260,11 @@ const styles = StyleSheet.create({
   wrapperSearch: {
     flex: 1,
     backgroundColor: COLORS.lightgrey,
-    borderRadius: dimen.normalizeDimen(8),
+    borderRadius: dimen.normalizeDimen(12),
     alignSelf: 'center',
     flexDirection: 'row',
     height: dimen.normalizeDimen(36),
-    paddingRight: dimen.normalizeDimen(8)
+    paddingRight: dimen.normalizeDimen(12)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The border radius and padding right of the `wrapperSearch` style have been increased. This will make the search box more rounded and provide more padding on the right side. Ensure that these changes fit well with the overall design.
```diff
- borderRadius: dimen.normalizeDimen(8),
- paddingRight: dimen.normalizeDimen(8)
+ borderRadius: dimen.normalizeDimen(12),
+ paddingRight: dimen.normalizeDimen(12)
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->